wordpress-mobile/WordPress-iOS
# Fix stories 9:16 export

Fixes https://github.com/tumblr/kanvas-ios/issues/62

## Testing

* Create a Story post
* Add landscape and portrait media
* Add text to the media
* Ensure that the media posted uploaded to the site is roughly 9:16. (about 0.5625 when dividing the width by the height)
* Ensure text is positioned as expected

## Regression Notes
1. Potential unintended areas of impact

Anything related to Story media export.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Tested posting stories with a variety of media sizes and types.

3. What automated tests I added (or what prevented me from doing so)

Unfortunately, the OpenGL rendering will not work in the simulator so we cannot test this.

PR submission checklist:
